schema_based_on_diagram.sql

```diff
@@ -0,0 +1,70 @@
+CREATE TABLE patients (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(250),
+    date_of_birth DATE
+)
+
+CREATE TABLE treatments (
+    id SERIAL PRIMARY KEY,
+    type VARCHAR(250),
+    name VARCHAR(250)
+)
+
+CREATE TABLE medical_histories (
+    id SERIAL PRIMARY KEY,
+    admitted_at TIMESTAMP,
+    patient_id INTEGER,
+    treatment_id INTEGER,
+    CONSTRAINT fk_patients FOREIGN KEY (patient_id) REFERENCES patients(id),
+    CONSTRAINT fk_treatments FOREIGN KEY (treatment_id) REFERENCES treatments(id),
```

Comment:
- [ ] the Foreign key for treatments shouldn't be set in the medical_histories table, this connection is already created in the Join table called medical_histories_treatments.